test/helpers/deploy/deploy_resources.go

```diff
@@ -214,6 +214,7 @@ func KonnectGatewayControlPlaneType(typ sdkkonnectcomp.CreateControlPlaneRequest
 			panic(fmt.Errorf("%T does not implement KonnectGatewayControlPlane", obj))
 		}
 		cp.Spec.CreateControlPlaneRequest.ClusterType = &typ
+		cp.Spec.CloudGateway = lo.ToPtr(false)
```

Comment:
This shouldn't be required. https://github.com/Kong/kubernetes-configuration/pull/343 should fix it.

Reply:
Cool, are you going to release an rc?

Reply:
I tested locally with the branch from mentioned PR and it works as expected

Reply:
> Cool, are you going to release an rc?

Yes, PTAL https://github.com/Kong/kubernetes-configuration/pull/344

Reply:
https://github.com/Kong/kubernetes-configuration/releases/tag/v1.2.0-rc.2